packages/components/src/badge/index.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import clsx from 'clsx';
+
+/**
+ * WordPress dependencies
+ */
+import { info, caution, error, published } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import type { BadgeProps } from './types';
+import Icon from '../icon';
+
+function Badge( {
+	className,
+	as: Component = 'div',
+	context = 'neutral',
```

Comment:
Though I do understand the benefits of using "neutral" in this context, I am leaning toward naming the default variant `"default"` 🤔 Otherwise I would have to look at the docs in order to know whether "neutral" is the default or not. What do you think?

For what it's worth, I think all our other components default to either `"default"` or `undefined`.